common/views/components/GlobalContextProvider/GlobalContextProvider.tsx

```diff
@@ -0,0 +1,92 @@
+import { GetServerSidePropsContext } from 'next';
+import { createContext, FunctionComponent, ReactNode } from 'react';
+import OpeningTimesContext from '../OpeningTimesContext/OpeningTimesContext';
+import GlobalAlertContext from '../GlobalAlertContext/GlobalAlertContext';
+import PopupDialogContext from '../PopupDialogContext/PopupDialogContext';
+import TogglesContext from '../TogglesContext/TogglesContext';
+import { parseCollectionVenues } from '../../../services/prismic/opening-times';
+
+export type GlobalContextData = {
+  toggles: any;
+  globalAlert: any;
+  popupDialog: any;
+  openingTimes: any;
+};
+
+type Props = {
+  value: GlobalContextData;
+  children?: ReactNode;
+};
+
+const defaultValue = {
+  toggles: {},
+  globalAlert: null,
+  popupDialog: null,
+  openingTimes: null,
+};
+
+export type WithGlobalContextData = {
+  globalContextData: GlobalContextData;
+};
+
+const Context = createContext<GlobalContextData>(defaultValue);
+const GlobalContextProvider: FunctionComponent<Props> = ({
+  value,
+  children,
+}: Props) => {
+  const parsedOpeningTimes = parseCollectionVenues(value.openingTimes);
+  return (
```

Comment:
Would globalInfoBarContext need to be included in the GlobalContextProvider also ?

Reply:
[I've added it to the `PageLayout`](https://github.com/wellcomecollection/wellcomecollection.org/pull/5773/commits/95412d40cd0cb000c34149ea3f6cb59f340b6f47) - sorry for not spotting the code drift.